extensions-core/protobuf-extensions/src/main/java/org/apache/druid/data/input/protobuf/ProtobufInputRowParser.java

```diff
@@ -127,49 +127,59 @@ void initDescriptor()
 
   private Descriptor getDescriptor(String descriptorFilePath)
   {
-    InputStream fin;
-
-    fin = this.getClass().getClassLoader().getResourceAsStream(descriptorFilePath);
+    InputStream fin = this.getClass().getClassLoader().getResourceAsStream(descriptorFilePath);
```

Comment:
This inputstream wasn't being closed.

Reply:
This was actually closed in `DynamicSchema.parseFrom()` which is a bad pattern.

Reply:
nit: can we use the `try-with-resources` instead of `CloseQuiently.close()`? I guess we can do if we extract creating `fin` as a method.